examples/ensemble/plot_adaboost_hastie_10_2.py

```diff
@@ -88,6 +108,20 @@
 for i, y_pred in enumerate(ada_real.staged_predict(X_train)):
     ada_real_err_train[i] = zero_one_loss(y_pred, y_train)
 
+# %%
+# Plotting the results
+# --------------------
+# Finally, we plot the train and test errors of our baselines
+# and of the discrete and real AdaBoost classifiers
+
+import matplotlib.pyplot as plt
+
+fig = plt.figure()
+ax = fig.add_subplot(111)
+
+ax.plot([1, n_estimators], [dt_stump_err] * 2, "k-", label="Decision Stump Error")
+ax.plot([1, n_estimators], [dt_err] * 2, "k--", label="Decision Tree Error")
+
```

Comment:
In the figure below, we can remove the `color` argument each time.
We can use the default colour that should be more suited to colourblindness.

Reply:
Good point!

Reply:
I have to improvise a bit here since the default colors include both red and green, which is the most frequent type of color-blindness according to [matplotlib](https://matplotlib.org/stable/tutorials/colors/colormaps.html#id11)